ci-operator/config/shiftstack/ci/shiftstack-ci-main.yaml

```diff
@@ -28,13 +28,15 @@ tests:
   steps:
     env:
       CLUSTER_TYPE_OVERRIDE: openstack-vexxhost
+      IMAGES_TTL: 336h
       OPENSTACK_AUTHENTICATION_METHOD: password
     workflow: shiftstack-cleanup
 - as: cleanup-openstack-nfv
   interval: 6h
   steps:
     env:
       CLUSTER_TYPE_OVERRIDE: openstack-nfv
+      IMAGES_TTL: 336h
```

Comment:
why all these clouds need to have that specific clean up?
Can you share the exact jobs that are expected to have ORC images running?

Reply:
Right now the HCP e2e job only runs on VEXXHOST AFIK but I didn't want to create a snowflake so I added this everywhere for consistency. Happy to discuss this and change it.

Reply:
I thought we could set it up only on the cloud that is really needed, which is vexxhost. In case there any issue related to that, it would be isolated there.
There are other hypershift related jobs that run on mecha for example (hypershift-e2e-openstack-aws-csi-cinder), but it does not run the hypershift e2e test.
I don't have strong opinion on this topic, if we decide to do it in all of them, let's also add to az0.

Reply:
sure let me update it then